templates/patterns/timeline/timeline.ui_patterns.yml

```diff
@@ -0,0 +1,41 @@
+timeline:
```

Comment:
Please use the same data provided here https://v1--europa-component-library.netlify.com/ec/components/detail/ec-component-timeline to set the preview.